export const UserErrors = {
  invalidId: {
    message: 'Invalid Id',
  },
  notFound: {
    message: 'User not found',
  },
  nameExist: {
    message: 'Name already exist',
  },
};
export const commonErrors = {
  noMessage: {
    message: `Something went wrong`,
  },
  orgNotFound: {
    message: `Organization not assigned for this user`,
  },
  userNotFound: {
    message: `User not found`,
  },
  emailExist: {
    message: 'Email id already exist',
  },
  phoneExist: {
    message: 'Phone number already exist',
  },
};
